hash/hash_test.go

```diff
@@ -38,10 +38,10 @@ func ExampleChooseSubset_selectOne() {
 	ret := ChooseSubset(tasks, 1, "my-key1")
 	fmt.Println(ret.UnsortedList()[0])
 
-	ret = ChooseSubset(tasks, 1, "somthing/another-key")
+	ret = ChooseSubset(tasks, 1, "something/another-key")
 	fmt.Println(ret.UnsortedList()[0])
 	// Output: task3
-	// task1
+	// task2
```

Comment:
afaict, the change of the spelling of the subset changes how the hash plays around and thus which task one gets?

Someone will need to consider this much more carefully than me.